airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -267,6 +267,9 @@ class TIRunContext(BaseModel):
     Can either be a "decorated" dict, or a string encrypted with the shared Fernet key.
     """
 
+    """List of Xcom keys that need to be cleared."""
+    xcom_keys_to_clear: Annotated[list[str], Field(default_factory=list)]
```

Comment:
It's odd, but doc comments for a variable goes after the var
```suggestion
    xcom_keys_to_clear: Annotated[list[str], Field(default_factory=list)]
    """List of Xcom keys that need to be cleared."""
```

Reply:
Ah! I thought it was a preference.